app/browser/tabs.js

```diff
@@ -912,8 +912,16 @@ const api = {
           createProperties.parent_partition = ''
         }
       }
-      if (!isAutoDiscardable(createProperties)) {
+
+      // tabs are allowed to be initially discarded (unloaded) if they are regular tabs
+      // i.e not about: pages, and not pinned
+      // but the tab creation request can determine that ultimately by passing in
+      // `discarded: true`
+      if (createProperties.pinned || !isHttpOrHttps(createProperties.url)) {
         createProperties.discarded = false
+      }
+      // similarly, autoDiscardable will happen for regular tabs (not about: tabs)
```

Comment:
should have a note that this is temporarily disabled and link to the issue

Reply:
it's still in the separate function, perhaps I should just put that logic inline here too since there's only 1 usage and it's 1 line. thoughts?

Reply:
That seems fine. This is the problem with these github reviews sometimes because important code is often not visible. The original issue with `isAutoDiscardable` was missed for a similar reason

Reply:
@bridiver I inlined it for now. Normally I like the functional / semantic language of bundling that logic out, but it was causing confusion in this case.